playground/main.py

```diff
@@ -62,7 +63,14 @@ def text(self, text=None):
         )
 
 
-EXAMPLE_FILES = ["hello.spy", "bluefunc.spy", "point.spy", "array.spy"]
+# Auto-discover example files from pyscript.toml
+with open("pyscript.toml", "rb") as f:
+    config = tomllib.load(f)
+EXAMPLE_FILES = [
+    f"examples/{source}"
+    for source, dest in config.get("files", {}).items()
+    if dest == "examples/"
+]
```

Comment:
this works, but there might be an even easier way to do it.
This file runs inside pyodide in the browser and it sees the virtual emscripten filesystem: so it should be enough to just use `glob('examples/*.spy')`